ocs_ci/utility/utils.py

```diff
@@ -2558,6 +2558,25 @@ def ceph_health_resolve_crash():
     archive_ceph_crashes(ct_pod)
 
 
+def ceph_health_resolve_mon_slow_ops(health_status):
+    """
+    Fix ceph health issue with daemon crash
+    """
+    log.warning("Trying to fix the issue with mon slow ops by restarting mon pod")
+    mon_pattern = r"mon\.([a-z]) has slow ops"
+    match = re.search(mon_pattern, health_status)
+    mon_id = None
+    if match:
+        mon_id = match.group(1)
+        log.warning(f"Problematic MON ID with slow ops: {mon_id} will be restarted")
+    if mon_id:
+        from ocs_ci.ocs import ocp
```

Comment:
can we move this at the start of this function if it has circular dependency issue

Reply:
Yes we have circular dependency issue, and I prefer to keep it here, as in case we don't get mon_id we don't need to import as we will not need ocp module to be imported.